### PR TITLE
Control access to /api-key endpoints

### LINKED ISSUE #22: Add some level of access control to `/api-key` endpoints.
For now, we could simply use API Gateway's api key feature.

### PR DESCRIPTION
**Pull request by forevermatt**
Closes #22 

This also protects against the JSON-parsing blowing up (instead returning a useful error message) if a JSON string of "\r\n" is provided (which I apparently accidentally did at one point).

### CODE REVIEW

File: helpers/request-helper.js [hunk 1]
```diff
@@ -10,5 +10,9 @@ module.exports.getTotpHeaders = (headers = {}) => {
 };
 
 module.exports.getJsonData = (jsonString) => {
+  var isOnlyWhitespace = /^\s+$/.test(jsonString);
```
Comment (wcjr) [inline helpers/request-helper.js]: What do you think about this?
```
jsonString = jsonString || '{}'; // only necessary if you can't trust the incoming arg.  I'm guessing that event.body is never null though...

return JSON.parse(jsonString.trim() || '{}');
```

Comment (forevermatt) [reply helpers/request-helper.js]: Unfortunately, `event.body` _is_ `null` when no body was provided in the request.

Given that, though, I think your two-line version here would work. I'll do that.

Comment (wcjr) [reply helpers/request-helper.js]: ok, I appreciate the consideration

Comment (forevermatt) [reply helpers/request-helper.js]: Done in 64d8eb1c

**Pull request merged**